eventbus/jetstream/base/jetstream.go

```diff
@@ -42,7 +42,13 @@ func (stream *Jetstream) Init() error {
 		stream.Logger.Error(errStr)
 		return errors.New(errStr)
 	}
+	err = stream.CreateStream(mgmtConnection)
```

Comment:
Creating a streaming is very low efficient, it takes more than 1 second (this is something I saw). I understand most of the `create` is actually `update`, did you that went through fast? Do we need to query first?

Reply:
I'm open to that, but do you think the latency will be an issue for Argo-Events (since message volume is relatively low)? I guess this extra one second will only occur once per EventSource on start up and once per Sensor on start up and after leader election, right?

Reply:
Does CreateStream() basically block for one second until it's done? If so, I guess we could look at whether something could get missed entirely when say a new leader gets elected and this extra latency is happening. On the Sensor side, the Consumer has a durable name, so there shouldn't be any issue of missing incoming messages (unless it's a brand new Trigger with a brand new Durable name). On the EventSource side, is there potential to miss an incoming Event because we're busy doing this?

Reply:
Okay, that's a good example. Let me see about checking if the Stream exists first.

Reply:
added check